msi/GitVars.wxi

```diff
@@ -0,0 +1,8 @@
+<?xml version='1.0'?>
+<Include>
+    <?if $(sys.BUILDARCH)=x64 ?>
+    <?define SixtyFourBit=64 ?>
+    <?else?>
+    <?define SixtyFourBit ?>
+    <?endif?>
+</Include>
```

Comment:
Why not write out this definition through `release.sh`? It already knows the bitness.